server/src/test/java/io/crate/integrationtests/DDLIntegrationTest.java

```diff
@@ -772,41 +763,6 @@ public void testDropBlobTableIfExistsUnknownTable() throws Exception {
         assertThat(response).hasRowCount(0);
     }
 
-    @Test
-    public void testAlterShardsOfPartitionedTableAffectsNewPartitions() throws Exception {
```

Comment:
As far as I can tell ` testAlterNumberOfShards` in `PartitionedTableIntegrationTest` is covering the exact same case.